src/test/java/org/cactoos/scalar/ScalarEnvelopeTest.java

```diff
@@ -45,6 +47,22 @@ public void envelopeDelegatesCalls() {
         ).affirm();
     }
 
+    @Test
+    public void propagatesException() {
+        new Assertion<>(
+            "must not alter the exception thrown by original Scalar",
+            new ScalarEnvelope<Integer>(
+                () -> {
+                    throw new UnsupportedOperationException("ok");
+                }) {
```

Comment:
@andreoss Is there many ```{``` here, isn't it? Can you check it and correct the ident?

Reply:
@fabriciofx Extracted parameter to variable